installer/build.sh

```diff
@@ -0,0 +1,4 @@
+GOOS=linux GOARCH=amd64 go build -o ./dist/installer-linux-amd64
```

Comment:
Maybe prefix with "node-..." to make the binaries more self-descriptive?

Reply:
Changed.